.github/workflows/vulnerabilities.yml

```diff
@@ -0,0 +1,47 @@
+name: Vulnerabilities
+
+on: [pull_request]
+jobs:
+  vulnerabilities:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Rails application check
+        id: rails-check
+        run: |
+          if [ -f config/application.rb ]; then
+            echo "RAILS_APP=true" >> "$GITHUB_OUTPUT"
+          fi
+
+      - name: Brakeman
+        env:
+          RAILS_APP: ${{ steps.rails-check.outputs.RAILS_APP }}
+          # Brakeman has dropped support for Ruby 2, so we need to use an older version until we upgrade Ruby
+          BRAKEMAN_VERSION: ~> 5.0
+        run: |
+          if [ -z "$RAILS_APP" ]; then
+            echo "Not a Rails application"
+            exit 0
+          fi
+
+          echo "source 'https://rubygems.org'" > Gemfile.brakeman
+
+          if [ -n "$BRAKEMAN_VERSION" ]; then
+            echo "gem 'brakeman', '$BRAKEMAN_VERSION'" >> Gemfile.brakeman
+          else
+            echo "gem 'brakeman'" >> Gemfile.brakeman
+          fi
+
+          bundle install --gemfile Gemfile.brakeman
+
+          bundle exec --gemfile Gemfile.brakeman brakeman --exit-on-warn
```

Comment:
Just curious if creating the Gemfile was required for some reason instead of the following if bundler audit is installing the gem globally too:
```suggestion
          if [ -n "$BRAKEMAN_VERSION" ]; then
            gem install brakeman -v "'${BRAKEMAN_VERSION}'"
          else
            gem install brakeman
          fi

          brakeman --exit-on-warn
```

Reply:
I don't remember a rationale for using Bundler to manage these gems instead of installing them globally. Switching to global installation might be a simpler option, and I'm open to making that change.

Reply:
I might have been considering the use of pessimistic version operator. Please correct me if I'm mistaken, but I don't think gem install accommodates this.